### ISSUE TITLE
Objects that get spread into `properties` don't get read by the analyzer

### ISSUE THREAD

**Issue by KonnorRogers** [open-wc/custom-elements-manifest#245]
**Checklist**
- [x] Did you run the analyzer with the `--dev` flag to get more information?
- [x] Did you create a **minimal** reproduction in the [playground](https://custom-elements-manifest.netlify.app/)?

Playground: <https://custom-elements-manifest.netlify.app/?source=CmltcG9ydCB7IExpdEVsZW1lbnQgfSBmcm9tICdsaXQtZWxlbWVudCc7Cgpjb25zdCBmb3JtUHJvcGVydGllcyA9IHsKICAgIGZvbzogewogICAgICAgIHJlZmxlY3Q6IHRydWUsCiAgICAgICAgYXR0cmlidXRlOiAiZm9vIgogICAgfQp9CgpleHBvcnQgY2xhc3MgTXlFbGVtZW50IGV4dGVuZHMgTGl0RWxlbWVudCB7CiAgc3RhdGljIGdldCBwcm9wZXJ0aWVzICgpIHsKICAgIHJldHVybiB7CiAgICAgIC4uLmZvcm1Qcm9wZXJ0aWVzLAogICAgICBiYXI6IHtyZWZsZWN0OiB0cnVlfQogICAgfQogIH0KfQo%3D&library=litelement>

TLDR: When you do something like:

```js
const formProperties = { foo: {}}

static properties = {
  ...formProperties
}
```

The form properties never get added to the custom elements manifest.

**Expected behavior**

```json
          "members": [
            {
              "kind": "field",
              "name": "foo",
              "privacy": "public",
              "attribute": "foo",
              "reflects": true,
            },
          ]
```

Current behavior:

```json

**Expected behavior**

```json
          "members": [
            {
              "kind": "field",
              "name": "-",
              "privacy": "public",
              "attribute": "",
            },
          ]
```
```
